apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,65 @@
+
+ * [File handlers](https://developer.mozilla.org/docs/Web/Manifest/file_handlers):  In a PWA, the file_handlers member specifies the types of files the PWA can handle. When installed, the browser uses this to associate the PWA with specific file types at the OS level.
+ * [Launch handlers](https://developer.mozilla.org/docs/Web/API/Launch_Handler_API): In a PWA, the launch_handler field specifies how the PWA should be launched and navigated to. 
+ * [Protocol handlers](https://developer.mozilla.org/docs/Web/Manifest/protocol_handlers): In a PWA, protocol handlers are used to register the PWA to handle certain allowed protocols. A protocol is a string that allows clients to identify a resource on the web, such as http or mailto. When a PWA is installed, it can register to handle specific protocols, such as mailto or geo.
+ * [Share target](https://docs.pwabuilder.com/#/home/native-features?id=web-share-api):  A share target is a feature in a Progressive Web App (PWA) that allows the app to receive shared content from other apps. This is done by adding a share_target member to the web app manifest, which specifies how the app should handle shared data. It allows for seamless integreation of the PWA with the native sharing capabilities of the device. 
+ * [Shortcuts](https://docs.pwabuilder.com/#/home/native-features?id=shortcuts): Shortcuts in a Progressive Web App (PWA) allow users to quickly access specific features or pages within the app. These shortcuts can be added to the web app manifest and are displayed when a user long-presses or right-clicks on the app icon on their device.
+ * [Widgets](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets): Widgets are small applications that can be embedded within a larger application or operating system. They are designed to provide quick access to frequently used information or functionality. In the context of an app manifest, widgets are defined in the PWA manifest file using the widgets manifest member.
+
+## Insights from the data
+
+* It seems like  shortcuts are among the more popular features that developers like to use. Shortcuts are relatively easy to add, and are a quick way (or should a say, a shortcut), to add a feature that makes the PWA more app like. On the other hand, features such as share_target, file_handlers and widgets require more work to enable and are also only relevant for certain scenarios so it's understandable why they are not as popular. 
+
+* The number of PWAs with widgets might seem low, but it was actually a surprise to me as widgets are relatively a newer feature. Earlier this year, Facebook was one of the first big PWAs to add widgets to their manifest. Widgets are an easy way to increase engagement across different surfaces on an operating system. The uptick in widgets is a positive indicator of increasing awareness among developers. 
```

Comment:
Is Facebook implementing widgets public information? We should check to make sure

Reply:
They were featured at Build right? I'll ask around

Reply:
Got confirmation from facebook itself that they have no problems with being mentioned :)